src/developer-release-signer.md

```diff
@@ -0,0 +1,5 @@
+# Developer Release
+
+[The sBTC Developer Release](./sbtc-releases/sbtc-dev.md) is a minimum-viable testnet deployment of sBTC and was decided to be released without the without the complexity of adding a decentralized set of signers to validate sBTC deposits/withdrawals.
```

Comment:
~~the without~~

Reply:
fixed cbec4022241ad010c47e210512c1112d94883e68